### ISSUE TITLE
Screen froze

### ISSUE THREAD

**Issue by Leo-MapCreator** [haxball/haxball-issues#561]
Screen froze but the chat was working, it unfroze after some time

**Comment by Leo-MapCreator** [comment]
Everyone in the room felt that

**Comment by basro** [comment]
That sounds strange, please tell me if it happens again or if you find a way to reproduce it.

**Comment by Leo-MapCreator** [comment]
It lagged before happening that, i dont know how to reproduce it

**Comment by iamrenan** [comment]
Perhaps something in your code? It happens when I accidentally create a recursion.

**Comment by Leo-MapCreator** [comment]
It wasnt my room and it wasn't a bot room

**Comment by Mirac35** [comment]
It happen'd to me more than five times before when i was using old PSU. I believe your PSU gives signals. Sending more/less voltage to system/cpu can create disequlibrium. Or just voltage problem (can happen anytime but rarely).

**Comment by Leo-MapCreator** [comment]
@Mirac35 does that happen with everyone in the room? because it happened with everyone in it, we could chat but the screen was frozen

**Comment by Mirac35** [comment]
> 
> 
> @Mirac35 does that happen with everyone in the room? because it happened with everyone in it, we could chat but the screen was frozen

yup, you can experiment this in a way if you have  very low/old CPU on your machine by extract something from zipped archive while you are on your room. everyone will feel the lag and if your cpu really tat old it can freeze your computer and room. other people can feel this too.

**Comment by Leo-MapCreator** [comment]
it was not lag, and if it freezes your CPU and your room, no one would be able to chat like we did

**Comment by Mirac35** [comment]
No i wanted to give it that name as lag dude. I've exprienced this before this was what i wanted to tell you. On screen everything is freezed like you pressed P. But chat is alive. Isnt tat you are exprienced ?
